websites/O/OtakuEyes/metadata.json

```diff
@@ -0,0 +1,30 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"id": "297055773714808833",
+		"name": "Yù"
+	},
+	"service": "OtakuEyes",
+	"description": {
+		"en": "My own platform for watching animes",
+		"fr": "Mon site pour regarder des animes"
+	},
+	"url": "192.168.0.7",
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/wFmlFjg.png",
```

Comment:
[Logos](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md#logo) should be 512x512px.

Reply:
update logo done

Reply:
Make sure to mark these as resolved when you complete them.